features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/components/APIDetailsTopMenu.jsx

```diff
@@ -59,13 +66,52 @@ const styles = (theme) => ({
         textOverflow: 'ellipsis',
         whiteSpace: 'nowrap',
     },
+    downloadApi: {
+        display: 'flex',
+        flexDirection: 'column',
+        textAlign: 'center',
+        justifyContent: 'center',
+        cursor: 'pointer',
+        height: 70,
+        color: theme.palette.getContrastText(theme.palette.background.paper),
+    },
+    downloadApiFlex: {
+        display: 'flex',
+        flexDirection: 'column',
+    },
 });
 
+
 const APIDetailsTopMenu = (props) => {
     const {
         classes, theme, api, isAPIProduct, imageUpdate,
     } = props;
     const isVisibleInStore = ['PROTOTYPED', 'PUBLISHED'].includes(api.lifeCycleStatus);
+    /**
+ * The component for advanced endpoint configurations.
+ * @param {string} name The name of the
+ * @param {string} version Version of the API
+ * @param {string} provider Provider of the API
+ * @param {string} format Weather to recive files in YALM of JSON format
+ * @returns {zip} Zpi file containing the API directory.
+ */
+    async function exportAPI() {
```

Comment:
Can we remove async here?

Reply:
if it removed it will push a null object to  Utils.forceDownload function

Reply:
We don't use async usually. There should be a way to do it without using async.

Reply:
Check WSDL.jsx -> handleDownloadWSDLZip() and its usage

Reply:
Done